config/crds/cluster_v1alpha1_machine.yaml

```diff
@@ -15,6 +15,11 @@ spec:
     description: Machine status such as Terminating/Pending/Running/Failed etc
     name: Phase
     type: string
+  - JSONPath: .status.nodeRef.name
+    description: Node name associated with this machine
+    name: Node Name
```

Comment:
i think the name convention here is common to be PascalCase - e.g. `NodeName`
also `ProviderID` is already using that.